modules/ocf_mirrors/files/project/archlinuxcn/sync-archive

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh -eu
+
+/usr/local/bin/rsync-no-vanished -rtlH --delete-after --delay-updates --safe-links \
+	ucb-ocf@sync.repo.archlinuxcn.org::repo /opt/mirrors/ftp/archlinuxcn
```

Comment:
where do the credentials for this come from

Reply:
They sent me the password via email. It's an environment variable that is not suitable to be stored publicly on GitHub.

Reply:
This should go in the Puppet private share, I can explain to you how to do this at a later point